.github/workflows/R_lintr.yaml

```
@@ -0,0 +1,33 @@
+on:
+  push:
+    branches:
+      - master
+      - continuous_integration
+  pull_request:
+    branches:
+      - master
+      - continuous_integration
+
+name: R linter
+
+jobs:
+  lint:
+    runs-on: macOS-latest
+    
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    
+    steps:
+      - uses: actions/checkout@v2
+     
+      - uses: r-lib/actions/setup-r@master
+
+      - name: Install dependencies
+        run: |
+          install.packages('remotes')
+          remotes::install_cran("lintr")
+        shell: Rscript {0}
+
+      - name: Lint
+        run: lintr::lint_package(linters = lintr::with_defaults(object_name_linter = NULL))
```

Maybe if we do something like:
```suggestion
        run: |
          results = lintr::lint_package(linters = lintr::with_defaults(object_name_linter = NULL))
          if (length(results)) { print(results); quit(status=1) }
```
we will get the failing exit status and if we're lucky, a proper printout

---

I'll give this a go, thanks Joel!

---

The R linter is now coming up with a true negative alert! :)

---

However, I don't think this is due to the linting itself ... there is an error popping up:
```
##[error]Error: unexpected '/' in "/"
Execution halted
##[error]Process completed with exit code 1.
```

---

So, a false true negative?? :D

---

It seems Rscript doesn't like receiving two commands in one.

This a more R-like single statement, and might make it happy:
```
if ( length(lintr::lint_package(linters = lintr::with_defaults(object_name_linter = NULL)) -> results)) { print(results); quit(status=1) }
```

---

I've just pushed that change to this branch. I had mean to submit it to you for review first, @mortisflux, but it seems my fingers went into automatic.

---

The lint failed (for the correct reasons). No nice printout of the linting as in R Studio though ...